doc/onboarding/model/form.md

```diff
@@ -0,0 +1,30 @@
+# `form` State
+
+## Definition
+State of a form that includes its values but also :
+ - required field or
+ - ability to submit or not all the form.
+
+```
+{
+    <boolean> disabled: false
```

Comment:
At this time do we have a not-mandatory step ? If not, maybe it's not our concern for the moment.